unified-codes/libs/ui/src/lib/components/inputs/molecules/SearchBar.tsx

```diff
@@ -24,10 +25,16 @@ export const SearchBar: SearchBar = ({ input, onChange, onClear, onSearch }) =>
   return (
     <Grid container>
       <Grid item xs={11}>
-        <ClearInput fullWidth value={input} onChange={onChangeText} onClear={onClear} />
+        <ClearInput
+          fullWidth
+          label={label}
+          value={input}
+          onChange={onChangeText}
+          onClear={onClear}
+        />
       </Grid>
       <Grid item xs={1}>
-        <SearchButton fullWidth onClick={onClick} />
+        <SearchButton fullWidth onClick={onClick} style={{ marginTop: 15 }} />
```

Comment:
Also - not a criticism but me wondering when it's good to put styles where 😕 . Inline for small things?

Reply:
I think inline is not good, no idea who did this!!